Code/Framework/AzToolsFramework/Tests/Prefab/PrefabInstanceDomGeneratorTests.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Prefab/PrefabInstanceDomGeneratorTestFixture.h>
+
+namespace UnitTest
+{
+    using PrefabInstanceDomGeneratorTests = PrefabInstanceDomGeneratorTestFixture;
+
+    TEST_F(PrefabInstanceDomGeneratorTests, GenerateInstanceDomDescendantOfFocusedOrRootSucceeds)
```

Comment:
nit: The word 'succeeds' is repetitive and can be removed to make the names smaller. If you remove it, consider renaming functions this way if it makes sense : GenerateInstanceDomOfDescendantWithFocusOnLevel

Reply:
Agreed and renamed.